schema.json

```diff
@@ -0,0 +1,2729 @@
+                        "@mapping": "takeshape:local:ProductVariantMetadata.HYbvCdWeB"
+                    }
+                }
+            }
+        },
+        "CollectionMetadata": {
+            "id": "x-p59adya",
+            "name": "CollectionMetadata",
+            "title": "Collection Metadata",
+            "description": "Extended metadata about a collection",
+            "workflow": "default",
+            "model": {
+                "type": "multiple"
+            },
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "collectionId": {
+                        "type": "string",
+                        "minLength": 0,
+                        "description": "",
+                        "title": "Collection ID",
+                        "@mapping": "takeshape:local:CollectionMetadata.P7w-kg68D"
+                    },
+                    "externalCollectionTitle": {
+                        "type": "string",
+                        "minLength": 0,
+                        "description": "",
+                        "title": "External Collection Title",
+                        "@mapping": "takeshape:local:CollectionMetadata.3ASw_SgnA"
+                    }
+                }
+            }
+        }
+    },
+    "workflows": {},
+    "forms": {
+        "ProductMetadata": {
+            "default": {
+                "properties": {
+                    "productId": {
+                        "widget": "singleLineText"
+                    },
+                    "externalProductionSource": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "productId",
+                    "externalProductionSource"
+                ]
+            }
+        },
+        "ProductVariantMetadata": {
+            "default": {
+                "properties": {
+                    "productVariantId": {
+                        "widget": "wysiwygSingleLineText"
+                    },
+                    "additionalPackingInstructions": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "productVariantId",
+                    "additionalPackingInstructions"
+                ]
+            }
+        },
+        "CollectionMetadata": {
+            "default": {
+                "properties": {
+                    "collectionId": {
+                        "widget": "singleLineText"
+                    },
+                    "externalCollectionTitle": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "collectionId",
+                    "externalCollectionTitle"
+                ]
+            }
+        }
+    },
+    "services": {
+        "takeshapetestsite": {
```

Comment:
We should find/replace `takeshapetestsite` with `my-shopify-shop`. This name is usually derived from the shopname but doesn't have to be when using a pattern. I think it makes sense to use and id that is easy to understand.